### DISCUSSION TITLE
Offer/suggest to install and setup a dotfile manager?

### DISCUSSION THREAD

**Discussion by jtagcat** [archlinux/archinstall#349] [Ideas]
The home directory can get messy and unmanaged. Starting with a manager is much easier than to adopt one later.

This is quite opinionated and can be viewed as junk in the installer (unneeded step). Where will the line be drawn? Should there be a 'do you want to view additional opinionated suggestions?' at the end, where 'first/fresh' install steps could be shoved under.

**Comment by Torxed** [comment]
So, the guided installer should to the best of it's ability follow the [Installation Guide](https://wiki.archlinux.org/index.php/Installation_guide). Any optional changes should be added to profiles where things like post-configuration stuff may live.

There's some wiggle room to this, ofc.. But in general this is the principle we try to follow.
